src/containers/genres/js/genres.js

```diff
@@ -22,25 +23,31 @@ function uncheck(event) {
 function storeGenreOptions(event) {
     event.preventDefault();
 
-    var yesGenreArray = [];
-    var noGenreArray = [];
+    var checked = document.querySelectorAll("input:checked")
 
-    for (var i = 0; i < inputYes.length; i++) {
-        if (inputYes[i].checked === true) {
-            yesGenreArray.push(inputYes[i].dataset.index);
-        };
-    };
+    if (checked.length === 0) {
+        alert("Please select at least one genre")
+    } else {
+        var yesGenreArray = [];
+        var noGenreArray = [];
 
-    for (var i = 0; i < inputNo.length; i++) {
-        if (inputNo[i].checked === true) {
-            noGenreArray.push(inputNo[i].dataset.index);
+        for (var i = 0; i < inputYes.length; i++) {
+            if (inputYes[i].checked === true) {
```

Comment:
inputYes[I].checked, by definition it is going to return true or false, so you can write it as:
```
if (inputYes[i].checked) { 
//do something
}
```
same in line 41.